pkg/controller/postgres/postgres_controller.go

```diff
@@ -110,12 +115,18 @@ func (r *ReconcilePostgres) Reconcile(request reconcile.Request) (reconcile.Resu
 		return reconcile.Result{}, nil
 	}
 
+	postgresClusterIP := "0.0.0.0"
```

Comment:
I think that this `postgresClusterIP` variable will have some value always from `postgresService.Spec.ClusterIP` (line 123)

Reply:
you're right